experimental/rules-keeper/store/aspect-build/rules_ts/versions/v0.1.0/metrics/version_stats.metadata

```diff
@@ -0,0 +1,4 @@
+update_time:  {
+  seconds:  1677692269
+  nanos:  122313000
+}
```

Comment:
For these METADATA files, is the update_time the time that the data was pulled? Is it necessary to store at the resolution of nanoseconds?

Reply:
Not necessary, but it comes directly from google.golang.org/protobuf/types/known/timestamppb. So I'd rather leave it as it is, instead of writing more code to truncate it.